src/reducers/gameReducer.js

```diff
@@ -1,29 +1,28 @@
 import * as types from '../actions/actionTypes';
 
 const initialState = {
-  isLoggedIn: false,
-  codeChallenge: `let counter = 0;
-for (let i = 0; i < test.length; i++) {
-  counter += 1;
-}`,
+  codeChallenge: 'let counter = 0;<br/>for (let i = 0; i < test.length; i++) {<br/>&nbsp;&nbsp;counter += 1;<br/>}',
```

Comment:
Shouldn't we be getting the challenges from the DB ?

Reply:
We will start fetching from the db once all the logic is correct and linked up